.travis.yml

```diff
@@ -13,7 +14,10 @@ install:
   - pip install beautifulsoup4 lektor pygments requests
 
 script:
-  - "python scripts/check_links.py"
+  # Only run link checks when it is a PR, not when merging on lektor branch
+  - if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then
+      python scripts/check_links.py ;
```

Comment:
If there are errors in CI, don't we want to make sure it _isn't_ deployed? Whats' the logic for skipping this?

Reply:
Well that was the logic but i think sometimes lnks break and having a couple of broken links should not prevent a deploy.... even more if the change and the break is unrelated.